fuel/app/migrations/001_create_areas.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Fuel\Migrations;
+
+class Create_areas
+{
+	public function up()
+	{
+		\DBUtil::create_table('areas', array(
+			'id' => array('constraint' => 11, 'type' => 'int', 'auto_increment' => true, 'unsigned' => true),
+			'code' => array('type' => 'text'),
+			'name' => array('type' => 'text'),
+			'country' => array('type' => 'text'),
+			'country_name' => array('type' => 'text'),
+			'city' => array('type' => 'text'),
+			'city_name' => array('type' => 'text'),
```

Comment:
@Hung-Phan-Mulodo , pls replace code, name, country, country_name, city, city_name from text to varchar(length) type.